### PR TITLE
Make repo config use pyproject.toml

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: changebot/github/github_api.py [hunk 1]
```diff
@@ -108,7 +109,7 @@ def get_file_contents(self, path_to_file):
         contents_base64 = response.json()['content']
         return base64.b64decode(contents_base64).decode()
 
-    def get_user_config(self, path_to_file='.astropybot.yml',
+    def get_user_config(self, path_to_file='pyproject.toml',
```
Comment (bsipocz) [inline changebot/github/github_api.py:112]: please update the docstrings, too

**Review by bsipocz — APPROVED** [review]
Looks good to me given you'll update the docstrings, too.

File: changebot/github/github_api.py (hunk 1 above)
Comment (Cadair) [reply changebot/github/github_api.py:112]: done

**Comment by pllim** [comment]
This broke a test on my own test repo that used the old YAML file -- https://github.com/pllim/playpen/blob/master/.astropybot.yml

Which makes me think... Are there random repos out there who use this old format? How would I rewrite my YAML to this new TOML? Do we need to maintain backward compatiblity?

**Comment by bsipocz** [comment]
@pllim - I think we're fine, your project is the only one we would break...

https://github.com/search?q=filename%3A.astropybot.yml&type=Code

**Comment by pllim** [comment]
OK then. I added the TOML thingy to my repo and restarted the test.

**Comment by pllim** [comment]
OK nvm, need to restart again when API limit timer wears off.

**Comment by pllim** [comment]
API limit error is gone, but now it is failing anyway. Turns out it is not grabbing stuff from my test repo, but from this repo itself. I suspect the failure would go away after merge.

**Review by pllim — APPROVED** [review]
LGTM

**Pull request merged**